### PR TITLE
Move positional params in e2e-test CI script and minor CI fixes

### PR DESCRIPTION
**Pull request by antgamdia**
### Description of the change

This PR fixes an issue with the vars passed to the e2e script. In GKE we are not testing the multicluster scenario, so the there were some missing positional params.

### Benefits

The CI will work again in GKE.

### Possible drawbacks

N/A

### Applicable issues

  - rel #2376

### Additional information

Note the changes are being tested in the [test-changes-ci](https://github.com/kubeapps/kubeapps/tree/test-changes-ci) branch.

### CODE REVIEW

**Review by andresmgot — APPROVED** [review]
LGTM

File: integration/use-cases/02-create-private-registry.js [hunk 1]
```diff
@@ -2,7 +2,11 @@ const axios = require("axios");
 const utils = require("./lib/utils");
 
 test("Creates a private registry", async () => {
-  var token = process.env.USE_MULTICLUSTER_OIDC_ENV ? undefined : process.env.ADMIN_TOKEN;
+  var token =
+    process.env.USE_MULTICLUSTER_OIDC_ENV === true ||
+      process.env.USE_MULTICLUSTER_OIDC_ENV === "true"
```
Comment (andresmgot) [inline integration/use-cases/02-create-private-registry.js:7]: an env var is always going to be a string

File: script/e2e-test.sh [hunk 2]
```diff
@@ -165,38 +165,18 @@ installOrUpgradeKubeapps() {
     # Install Kubeapps
     info "Installing Kubeapps..."
     kubectl -n kubeapps delete secret localhost-tls || true
-    
+
     helm upgrade --install kubeapps-ci --namespace kubeapps "${chartSource}" \
       ${invalidateCacheFlag} \
       "${img_flags[@]}" \
       "${@:2}" \
+      "${multiclusterFlags[@]+"${multiclusterFlags[@]}"}" \
```
Comment (andresmgot) [inline script/e2e-test.sh:173]: why is not this line just?:

```
      "${multiclusterFlags[@]}" \
```

File: integration/use-cases/02-create-private-registry.js (hunk 1 above)
Comment (antgamdia) [reply integration/use-cases/02-create-private-registry.js:7]: Ooops, should have checked it, sorry. Changing it in the #2651

File: script/e2e-test.sh (hunk 2 above)
Comment (antgamdia) [reply script/e2e-test.sh:173]: See https://stackoverflow.com/a/61551944/4178335 and #2651

**Pull request merged**